setup.py

```diff
@@ -91,6 +91,7 @@ def run(self):
     "dremio": ["pyarrow >= 5.0.0"],
     "exasol": ["pyexasol >= 0.16.1"],
     "sodasql": ["soda-sql >= 2.0.0b25"],
```

Comment:
This should not be added .. the soda-sql deps are transitive from soda-spark

Reply:
Hey @vijaykiran 😊 
Thanks for the suggestion!
I tried the change, but still getting the same error https://app.circleci.com/pipelines/github/PrefectHQ/prefect/10298/workflows/1a2da562-1dbf-40be-bd14-e7573bbb3f42/jobs/136120